libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/internal/kapt/KaptGenerateStubsTask.kt

```diff
@@ -85,18 +87,16 @@ open class KaptGenerateStubsTask : KotlinCompile() {
                 !stubsDir.isParentOf(source) &&
                 !generatedSourcesDir.isParentOf(source)
 
-    private val compileKotlinArgumentsContributor by project.provider {
-        kotlinCompileTask.compilerArgumentsContributor
+    @get:Internal
+    override val compilerArgumentsContributor: CompilerArgumentsContributor<K2JVMCompilerArguments> by lazy {
+        KotlinJvmCompilerArgumentsContributor(KotlinJvmCompilerArgumentsProvider(this), setOf(KAPT_SUBPLUGIN_ID))
```

Comment:
nit: Add a comment why this plugin is filtered out?

Reply:
Removed in the latest commit.